src/screens/BeersFavourite/FavouriteBeers.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { PureComponent } from 'react';
+import Heading from '../../shared/components/Heading/Heading';
+import Beer from '../../shared/components/Beer/Beer';
+import './styles.scss';
+import DeleteButton from '../../shared/components/DeleteButton/DeleteButton';
+import { getItemFromLocalStorage, saveItemInLocalStorage } from '../../services/localStorage'; 
+
+class FavouriteBeersComponent extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      beers: [],
+    }
+  }
+  
+  componentDidMount = () => {
+    const beers = getItemFromLocalStorage('beer') || [];
```

Comment:
Naming indicates that you retrieve single beer from local storage and assign it to array of beers?

Reply:
Fixed!